compiler/ballerina-lang/src/main/java/org/ballerinalang/compiler/plugins/AbstractCompilerPlugin.java

```diff
@@ -85,6 +85,6 @@ public void process(EndpointNode endpointNode, List<AnnotationAttachmentNode> an
     }
 
     @Override
-    public void codeGenerated(Path binaryPath) {
+    public void codeGenerated(String packageID, Path binaryPath) {
```

Comment:
Use PackageID instead of a string, because PackageID is more useful than a string.